specification/src/specification/error-handling.md

```diff
@@ -12,6 +12,7 @@ Data connectors should use standard HTTP error codes to signal error conditions
 | 409 | Conflict | The request could not be handled because it would create a conflicting state for the data source - for example, a mutation might fail because a foreign key constraint was not met. |
 | 500 | Internal Server Error | The request could not be handled because of an error on the server |
 | 501 | Not Supported | The request could not be handled because it relies on an unsupported [capability](capabilities.md). _Note_: this ought to indicate an error on the _caller_ side, since the caller should not generate requests which are incompatible with the indicated capabilities. |
+| 502 | Bad Gateway | The request could not be handled because an unstream service was unavailable or returned an unexpected response, e.g. a connection to a database server failed |
```

Comment:
Minor typo:
```suggestion
| 502 | Bad Gateway | The request could not be handled because an upstream service was unavailable or returned an unexpected response, e.g., a connection to a database server failed |
```